src/kinetics/Reaction.cpp

```diff
@@ -257,6 +348,40 @@ ChebyshevReaction::ChebyshevReaction(const Composition& reactants_,
 {
 }
 
+void ChebyshevReaction::getParameters(AnyMap& reactionNode) const
```

Comment:
Would it make sense to pass the unit system as a parameter? Presumably the destination is known at the time when the process starts?

Reply:
mostly moot as I believe YamlWriter converts later.